index.html

```diff
@@ -1,48 +1,249 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio skeleton</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Playfair+Display&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+  </head>
 
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>My Portfolio skeleton</title>
-  <link rel="stylesheet" href="style.css">
-  <link href="https://fonts.googleapis.com/css?family=Playfair+Display&display=swap" rel="stylesheet" />
-  <link href="https://fonts.googleapis.com/css?family=Poppins&display=swap" rel="stylesheet" />
-</head>
+  <body>
+    <main>
+      <header>
+        <nav class="navbar">
+          <div class="menu-ham">
+            <img src="././images/icon-menu.svg" alt="ham icon menu" />
+          </div>
+        </nav>
+      </header>
+      <div class="container-box"></div>
 
-<body>
-  <main>
-    <header>
-      <nav class="navbar">
-        <div class="menu-ham">
+      <section class="intro">
+        <div>
+          <h1 class="name">Charlie Giorgania</h1>
+          <p class="intro-text">
+            Hello! I am a software developer!<br />
+            I can help you build a product, feature or website.<br />
+            Take a look of my works.<br />If you like what you see and have a<br />
+            project you need coded, don’t hesitate and contact me.
+          </p>
+        </div>
+        <div class="start-collab-box">
+          <button type="button">Start collaboration</button>
+        </div>
+        <div class="art"></div>
+      </section>
+    </main>
+    <section class="projects-grid">
+      <div class="projects-title">
+        <h2>Projects</h2>
+      </div>
 
-          <img src="images/icon-menu.svg" alt="ham icon menu" />
+      <img src="./images/projects-pic-1.svg" alt="workspace zenith" />
+
+      <div class="skills-box">
+        <h3 class="project-name">Project name goes here</h3>
+        <div class="skills-style">
+          <button type="button" class="button-project">HTML/CSS</button>
+          <button type="button" class="button-project">Ruby on Rails</button>
+          <button type="button" class="button-project">Javascript</button>
+        </div>
+        <div class="project-btn">
+          <button>See this project</button>
+          <span>
+            <img src="./images/icon-arrow-right.svg" alt="icon arrow"
+          /></span>
         </div>
```

Comment:
- [ ] Please from the project template you choose, this first block is your first project. Therefore, make sure that the content of the project is aligned with the flexbox. ( suggestion: Create a `<div>` block)
- Please this needs to be applied to each one of the 6 projects.